src/VirtoCommerce.NotificationsModule.Core/Services/INotificationLayoutService.cs

```diff
@@ -0,0 +1,9 @@
+namespace VirtoCommerce.NotificationsModule.Core.Services
+{
+    /// <summary>
+    /// The service has a logic query/save of notification layouts to Database
+    /// </summary>
+    public interface INotificationLayoutService
```

Comment:
INotificationLayoutSearchService should be inherited from ICrudService<...> Ask @artem-dudarev